Atomic/syscontainers.py

```diff
@@ -141,6 +142,18 @@ def pull_image(self):
         return
 
     def install_user_container(self, image, name):
+        bwrap_oci_found = True
+        try:
+            util.check_call([BWRAP_OCI_PATH, "--version"], stdout=DEVNULL)
+        except util.FileNotFound:
+            bwrap_oci_found = False
+
+        if not bwrap_oci_found:
+            raise ValueError("Cannot install the container: bwrap-oci is needed to run user containers")
+
+        if not "--user" in str(util.check_output(["systemctl", "--help"], stdin=DEVNULL, stderr=DEVNULL)):
+            raise ValueError("Cannot install the container: systemctl does not support --user")
+
```

Comment:
the advantage of doing it here is that Atomic.syscontainers can be used as a library from other applications and this check would still be done